kolibri/core/assets/src/content-renderer.vue

```diff
@@ -14,25 +11,40 @@
 
   module.exports = {
     props: {
-      channelId: String,
-      contentId: String,
-      kind: String,
-      extension: String,
-      title: String,
       contentData: {
         type: Object,
-        default: () => ({ url: '#' }),
+        default: () => ({ pk: 0 }),
```

Comment:
Fine with these, except `extension` which relies on logic about the files themselves, and I would rather keep the file preference logic encapsulated in the content renderer, and not have it leak out into other parts of the app.